webpack/scenes/ContentViews/Details/Versions/ContentViewVersions.js

```diff
@@ -64,7 +64,7 @@ const ContentViewVersions = ({ cvId }) => {
       errata_counts: errataCounts,
     } = cvVersion;
     return [
-      { title: <a href={urlBuilder(`content_views/${cvId}/versions/${versionId}`, '')}>{__('Version ')}{version}</a> },
+      { title: <a href={urlBuilder(`labs/content_views/${cvId}#versions?subContentId=${versionId}`, '')}>{__('Version ')}{version}</a> },
```

Comment:
out of curiosity, why do we go this route of using subContentId instead using a url like  /content_views/CV_ID/versions/VERSION_ID ?

Reply:
or even if we're using a param, using version_id instead of subContentId, what would we do if we needed a 2nd subContetnId? (I can't think of any examples right now that would need that, more just curious)

Reply:
This is using pre-existing routing methods that I will likely change in an upcoming PR. 
This route will more than likely change to that exact address: /content_views/CV_ID/versions/VERSION_ID#whateverCurrentTabIAmOn